src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/xresilience/RetriesSpec.groovy

```diff
@@ -353,7 +350,8 @@ class RetriesIsolatedSpec extends HealthCheckSpecification {
     @Tags([ISL_RECOVER_ON_FAIL, LOW_PRIORITY])
     def "System does not retry after global timeout for reroute operation"() {
         given: "A flow with ability to reroute"
-        def swPair = switchPairs.all().nonNeighbouring().random()
+        def swPair = switchPairs.all().nonNeighbouring().switchPairs
+                .find { it.src.dpId.toString().contains("03") && it.dst.dpId.toString().contains("07")}
```

Comment:
hardcoded switches to get stability, the main intention is to catch Global timeout and system recovery(this test was stable when reroute timeout was 30 sec, but after increasing that value to 45, the instability occurred as the rest of the timeouts were not changed)